src/tactic/suggest.lean

```diff
@@ -309,13 +317,21 @@ For performance reasons `suggest` uses monadic lazy lists (`mllist`). This means
 `suggest` might miss some results if `num` is not large enough. However, because
 `suggest` uses monadic lazy lists, smaller values of `num` run faster than larger values.
 -/
-meta def suggest (n : parse (with_desc "n" small_nat)?) : tactic unit :=
-do L ← tactic.suggest_scripts (n.get_or_else 50),
-  when (¬ is_trace_enabled_for `silence_suggest)
-    (if L.length = 0 then
+meta def suggest (n : parse (with_desc "n" small_nat)?)
+  (hs : parse simp_arg_list) (attr_names : parse with_ident_list) (opt : by_elim_opt := { }) :
+  tactic unit :=
```

Comment:
You'll need to update the docs to explain the new arguments. Also on `library_search`.

Reply:
I've added,

"Once `library_search` applies a lemma from the library it attempts to close any remaining goals using `solve_by_elim`. Thus any inputs accepted by `solve_by_elim` are also accepted by `suggest`. This allows the user to add lemmas to the list of hypotheses, either as a list of lemmas or as a list of attribute names."

To the doc string for `library_search`, and

"Because `suggest` attempts to close any remaining goals using `solve_by_elim`, any inputs accepted by `solve_by_elim` are also accepted by `suggest`. This allows the user to add lemmas to the list of hypotheses, either as a list of lemmas or as a list of attribute
names."

to the doc string for `suggest`.

Is this ok or does it need more work?